de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java

```diff
@@ -669,23 +670,27 @@ public void doPeriodicSessionSave() {
     }
 
 
-   void postBootstrap(final Panel panel) {
+    void postBootstrap(final Panel panel) {
         setBrowserContextMenuEnabled(deProperties.isContextClickEnabled());
         messagePoller.start();
         initKBShortCuts();
         panel.add(view);
         processQueryStrings();
         getNotifications();
-   }
+        getSystemMessages();
+    }
 
-   @Override
-   public void getNotifications() {
-       messageServiceFacade.getRecentMessages(new InitializationCallbacks.GetInitialNotificationsCallback(view, appearance, announcer));
-       messageServiceFacade.getMessageCounts(new NewSysMessageCountCallback());
-   }
+    private void getSystemMessages() {
```

Comment:
This method name was a little confusing at first. Maybe it could be `getSystemMessageCounts`?